apps/server/src/modules/authorization/reference.loader.ts

```diff
@@ -31,6 +48,7 @@ export class ReferenceLoader {
 		this.repos.set(AllowedAuthorizationEntityType.School, { repo: this.schoolRepo });
 		this.repos.set(AllowedAuthorizationEntityType.Lesson, { repo: this.lessonRepo });
 		this.repos.set(AllowedAuthorizationEntityType.Team, { repo: this.teamsRepo, populate: true });
+		this.repos.set(AllowedAuthorizationEntityType.Submission, { repo: this.submissionRepo, populate: true });
```

Comment:
Submission repo do not support populate true, base repo also not. That is confusing and bring us to wrong implementations.

That is the reason why we want revert this changes. 
They are coming from external team.

Reply:
I removed the option for now.